src/java/org/apache/cassandra/auth/AuthenticatedUser.java

```diff
@@ -74,6 +97,22 @@ public RoleResource getPrimaryRole()
         return role;
     }
 
+    /**
+     * The mode of authentication used to authenticate this user, e.g. 'password', 'mtls', 'anonymous', etc.
+     */
+    public String getMode()
+    {
+        return mode;
+    }
+
+    /**
+     * {@link IAuthenticator}-contextual metadata about how the user was authenticated.
+     */
+    public Map<String, String> getMetadata()
```

Comment:
Primarily for serialization reasons.  I want to surface this in `system_views.clients`;  But I suppose in theory I could convert the map to a `<string,string>` when representing it in the table, so this could feasibly be `<string,object>`

Reply:
changed it to `<string, object>` :+1: